app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -362,7 +359,7 @@ class CtaViewModelTest {
     }
 
     @Test
-    fun whenRefreshCtaOnHomeTabAndSuppressWidgetCtaFeatureActiveThenReturnNullWhenTryngToShowWidgetInstructionsCta() = runBlockingTest {
+    fun whenRefreshCtaOnHomeTabAndSuppressWidgetCtaFeatureActiveThenReturnNullWhenTryingToShowWidgetInstructionsCta() = runBlockingTest {
         setSuppressHomeTabWidgetCtaFeature()
         setCovidCtaShown()
```

Comment:
This can be removed  I think.

Reply:
I was just fixing a typo on the test name, it already existed though, it said `Tryng` instead of `Trying`